lib/rack/utils.rb

```diff
@@ -551,6 +551,7 @@ def names
       428 => 'Precondition Required',
       429 => 'Too Many Requests',
       431 => 'Request Header Fields Too Large',
+      451 => 'Unavailable For Legal Reasons',
```

Comment:
Would it be wise to put a comment on here stating that the code isn't yet present on the IANA list as of the current date, in the unlikely event that someone tries to import from the CSV linked in the comment on [line 503](https://github.com/rack/rack/pull/985/files#diff-7c1a24d5b2fe58a6f925c7cacc6c55e7R503)?